docs/concepts/content-addressing.md

```diff
@@ -60,35 +60,40 @@ Check out ProtoSchool's [Anatomy of a CID](https://proto.school/anatomy-of-a-cid
 Converting a CID from v0 to v1 enables it to be represented in multibase encodings.
 The default for CIDv1 is the case-insensitive `base32`, but use of the shorter `base36` is encouraged for IPNS names to ensure same text representation on [subdomains](../how-to/address-ipfs-on-web.md#subdomain-gateway).
 
-### v0 to v1
+### With Kubo
 
-The built-in `ipfs cid format` command can be used from the command line:
+The [`ipfs cid format` command in Kubo](/reference/kubo/cli/#ipfs-cid-format) can be used from the command line:
 
 ```
 $ ipfs cid format -v 1 -b base32 QmbWqxBEKC3P8tqsKc98xmWNzrzDtRLMiMPL8wBuTGsMnR
 bafybeigdyrzt5sfp7udm7hu76uh7y26nf3efuylqabf3oclgtqy55fbzdi
 ```
 
-JavaScript users can also leverage the `toV1()` method provided by the [`cids`](https://www.npmjs.com/package/cids) library:
-```js
-const CID = require('cids')
-new CID('QmbWqxBEKC3P8tqsKc98xmWNzrzDtRLMiMPL8wBuTGsMnR').toV1().toString('base32')
-// → bafybeigdyrzt5sfp7udm7hu76uh7y26nf3efuylqabf3oclgtqy55fbzdi
-```
+### Programatically with TypeScript/JavaScript
+
+The [`multiformats` npm package](https://www.npmjs.com/package/multiformats) can be used to convert between CID versions, base representations, and access the binary representation:
+
+<iframe src="https://codesandbox.io/embed/working-with-cids-xrvqop?autoresize=1&fontsize=14&hidenavigation=1&theme=dark"
```

Comment:
Including dependency on external service that may dissapear at any time is a bit brittle :-1:

Reply:
Are you referring to https://codesandbox.io/? It's a pretty mature service so I think it may be worth considering the benefit vs cost of this.

Do you think that having an interactive/runnable example is useful? I'd argue that yes (even though this page may not be the right one for it and it should go into the guides)

Reply:
Summary of in-person discussion with @lidel:
The main problem with embedding codesandbox is that it won't work when the page is loaded locally without an internet connection.

While that's true, I think the benefits of having these interactive examples outweigh the cost of them not working without internet connection.